hydradx/tests/test_omnipool_agents.py

```diff
@@ -168,3 +168,38 @@ def test_omnipool_LP(omnipool: oamm.OmnipoolState):
     final_state = events[-1]
     if final_state.agents['agent'].holdings['HDX'] != 0 or events[-2].agents['agent'].holdings['HDX'] == 0:
         raise AssertionError('HDX not invested at the right time.')
+
+
+def test_agent_copy():
+    import random
+    init_agent = Agent(holdings={'HDX': 100, 'USD': 100}, share_prices={'HDX': 1, 'USD': 1})
+    def randomize_object(obj):
```

Comment:
This should be pulled out of the function, maybe it can stay in this file for now until/unless we will use it elsewhere.

Reply:
done. I do want to use it elsewhere because we can run the same test on other objects we want to fast copy.